packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -22,6 +22,7 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
     private _orderId?: string;
     private _isAPM?: boolean;
     private _isPaylater?: boolean;
+    private _isVenmo?: boolean;
```

Comment:
(optional)
I don't see any cases of usage _isVenmo as a global variable, because we are using it only in initialization method, therefore we can create a `const isVenmo` inside initialization method.